### PR TITLE
Content migration batch 9

### PR DESCRIPTION
**Pull request by mattletford**
Migrating content

https://developers.google.com/web/fundamentals/performance/audit/tools
https://web.dev/performance-audit-tools

https://developers.google.com/web/fundamentals/performance/critical-rendering-path/adding-interactivity-with-javascript
https://web.dev/critical-rendering-path-adding-interactivity-with-javascript

https://developers.google.com/web/fundamentals/performance/critical-rendering-path/analyzing-crp
https://web.dev/critical-rendering-path-analyzing-crp

https://developers.google.com/web/fundamentals/performance/critical-rendering-path/constructing-the-object-model
https://web.dev/critical-rendering-path-constructing-the-object-model

https://developers.google.com/web/fundamentals/performance/critical-rendering-path/measure-crp
https://web.dev/critical-rendering-path-measure-crp

@rachelandrew

### CODE REVIEW

File: src/site/content/en/blog/performance-audit-tools/index.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+You can keep an eye on Google Analytics
+[Site Speed](https://support.google.com/analytics/answer/1205784) to check how
+performance metrics correlate with business metrics. For example: 'how fast did the homepage load?'
+compared to 'did entry via the home page result in a sale?'
+<br><br>
+
+<figure>
+  {% Img src="image/C47gYyWYVMMhDmtYSLOWazuyePF2/jw1eTOeLv1S7jqrgQxlU.png", alt="Screenshot showing Google Analytics Site Speed", width="800", height="417" %}
+</figure>
+<br>
```
Comment (rachelandrew) [inline src/site/content/en/blog/performance-audit-tools/index.md:293]:
```suggestion
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]
See comments inline

**Comment by rachelandrew** [comment]
@mattletford this one still has a bunch of changes to tidy up so we can get it merged.

File: src/site/content/en/blog/performance-audit-tools/index.md [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+[event tracking](/analytics/devguides/collection/analyticsjs/events)
+for user actions such as button clicks and video plays. You may also want to implement
+[goal flow analysis](https://support.google.com/analytics/answer/2520139?ref_topic=1649581):
+the paths by which your users navigate towards 'conversions'.
+
+You can keep an eye on Google Analytics
+[Site Speed](https://support.google.com/analytics/answer/1205784) to check how
+performance metrics correlate with business metrics. For example: 'how fast did the homepage load?'
+compared to 'did entry via the home page result in a sale?'
+<br><br>
```
[the remark this answers is not in the document]
Comment (mattletford) [reply src/site/content/en/blog/performance-audit-tools/index.md]: @rachelandrew removed

File: src/site/content/en/blog/performance-audit-tools/index.md [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+</figure>
+
+## Test first and subsequent load performance
+
+[Lighthouse](/web/tools/lighthouse/),
+[WebPagetest](https://www.webpagetest.org/easy) and
+[Pagespeed Insights](/speed/pagespeed/insights/) are useful for analyzing speed, data cost and resource usage. WebPagetest will also check static-content
+caching, time to first byte, and if your site makes effective use of CDNs.
+
+{% Aside %}
```
[the remark this answers is not in the document]
Comment (mattletford) [reply src/site/content/en/blog/performance-audit-tools/index.md:205]: @rachelandrew converted to markdown

File: src/site/content/en/blog/performance-audit-tools/index.md [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+- Check the DevTools timeline for anything that blocks page load.
+
+If you're working from fast wifi or a fast cellular connection, test with [low bandwidth and high
+latency emulation](/web/tools/chrome-devtools/network-performance/network-conditions).
+Remember to test on mobile as well as desktop — some sites use UA sniffing to deliver different
+assets and layouts for different devices. You may need to test on actual hardware using
+[remote debugging](/web/tools/chrome-devtools/remote-debugging/),
+not just with device simulation.
+
+<div class="note">
```
[the remark this answers is not in the document]
Comment (mattletford) [reply src/site/content/en/blog/performance-audit-tools/index.md]: @rachelandrew converted to markdown

File: src/site/content/en/blog/performance-audit-tools/index.md [hunk 5]
```diff
@@ -0,0 +1,356 @@
...
+- Analytics
+- Metrics provided by servers and business systems
+- Screen and video recording
+- Manual tests
+
+Below you'll learn which approach is relevant for each type of audit.
+<br><br>
+
+{% Aside %}
+<strong>Images constitute by far the
```
[the remark this answers is not in the document]
Comment (mattletford) [reply src/site/content/en/blog/performance-audit-tools/index.md]: @rachelandrew converted to markdown

File: src/site/content/en/blog/performance-audit-tools/index.md [hunk 6]
```diff
@@ -0,0 +1,356 @@
...
+- Browser extensions
+- Online test applications
+- Emulation tools
+- Analytics
+- Metrics provided by servers and business systems
+- Screen and video recording
+- Manual tests
+
+Below you'll learn which approach is relevant for each type of audit.
+<br><br>
```
[the remark this answers is not in the document]
Comment (mattletford) [reply src/site/content/en/blog/performance-audit-tools/index.md]: @rachelandrew removed

File: src/site/content/en/blog/critical-rendering-path-constructing-the-object-model/index.md [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+  </body>
+</html>
+```
+
+[Try it](https://googlesamples.github.io/web-fundamentals/fundamentals/performance/critical-rendering-path/basic_dom.html)
+
+Let’s start with the simplest possible case: a plain HTML page with some text
+and a single image. How does the browser process this page?
+
+<img src="images/full-process.png" alt="DOM construction process">
```
[the remark this answers is not in the document]
Comment (mattletford) [reply src/site/content/en/blog/critical-rendering-path-constructing-the-object-model/index.md]: @rachelandrew image uploaded

File: src/site/content/en/blog/critical-rendering-path-adding-interactivity-with-javascript/index.md [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+</html>
+```
+
+[Try it](https://googlesamples.github.io/web-fundamentals/fundamentals/performance/critical-rendering-path/script.html)
+
+- JavaScript allows us to reach into the DOM and pull out the reference to the hidden span node; the node may not be visible in the render tree, but it's still there in the DOM. Then, when we have the reference, we can change its text (via .textContent), and even override its calculated display style property from "none" to "inline." Now our page displays "**Hello interactive students!**".
+
+- JavaScript also allows us to create, style, append, and remove new elements in the DOM. Technically, our entire page could be just one big JavaScript file that creates and styles the elements one by one. Although that would work, in practice using HTML and CSS is much easier. In the second part of our JavaScript function we create a new div element, set its text content, style it, and append it to the body.
+
+<img src="images/device-js-small.png"  alt="page preview">
```
[the remark this answers is not in the document]
Comment (mattletford) [reply src/site/content/en/blog/critical-rendering-path-adding-interactivity-with-javascript/index.md]: @rachelandrew image uploaded

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**